packages/slice-machine/sentry.client.config.ts

```diff
@@ -0,0 +1,27 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+import getConfig from "next/config";
+
+type SMNextConfig = { publicRuntimeConfig: { release: string } };
+
+const { publicRuntimeConfig } = getConfig() as SMNextConfig;
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+// TODO add `DEV_ENABLE_SENTRY` in Readme
+const initSentry =
+  process.env.NODE_ENV !== "development" || process.env.DEV_ENABLE_SENTRY;
```

Comment:
These variables might need to be injected in to the build through webpack, because process.env won't be available on the client.

Reply:
They're replaced at compilation time by webpack. I checked with the bundled version locally

Reply:
(this is gone in the last version of the PR)